front/src/common/api/generatedEditoastApi.ts

```diff
@@ -1479,6 +1490,19 @@ export type PostWorkSchedulesApiResponse =
 export type PostWorkSchedulesApiArg = {
   workScheduleCreateForm: WorkScheduleCreateForm;
 };
+export type PostWorkSchedulesProjectPathApiResponse =
+  /** status 201 Returns a list of work schedules whose track ranges intersect the given path */ {
+    end_date_time: string;
+    path_position_ranges: (number & number)[][];
```

Comment:
This syntax seems strange, shouldn't this be `[number, number][]`?

Reply:
it should, we can’t really control what’s generated

Reply:
Sounds like a bug in the Rust OpenAPI generator. It should use [`prefixItems`](https://json-schema.org/draft/2020-12/json-schema-core#section-10.3.1.1) instead of two identical items in `allOf`.

A workaround would be to use a struct instead of a tuple in the Rust types. (Maybe that's better anyways since it'd be more explicit what the two values are.)

I'd recommend searching/opening an issue on the Rust OpenAPI generator anyways.

Reply:
Seems to be https://github.com/juhaku/utoipa/issues/901.

Reply:
This might help? https://github.com/OpenRailAssociation/osrd/pull/8897

Reply:
Resolving to close here, better to fix this in next PR

Reply:
For information, the other PR got merged.